README.md

```diff
@@ -150,6 +150,20 @@ for a duplicate `v1` UUID to appear when generating UUIDs at a rate of 1M/second
 unlikely, [just like with `v4` UUIDs](#how-unique-are-v4-uuids) there is no practical guarantee
 that `v1` UUIDs are unique.
 
+### So v1 UUIDs potentially leak my MAC address and their creation timestamp?
+
+If implementations follow
+[the primary recommendations of RFC 4122](https://tools.ietf.org/html/rfc4122#section-4.1.6) then
+`v1` UUIDs would indeed leak the hardware MAC address of the machine where they are being created.
+[As discussed above](#but-arent-v1-uuids-better-because-they-are-guaranteed-to-be-unique) this
+would most likely not be the case in modern implementations where hardware MAC addresses are either
+unavailable (browser) or not unique (containers) but there are
```

Comment:
I would make this two sentences:

") this
would most likely not be the case in modern implementations where hardware MAC addresses are either
unavailable (browser) or not unique (containers)". However, there are rumors that the presence  of the MAC...